configs/policy.yml

```diff
@@ -0,0 +1,11 @@
+allowed:
```

Comment:
According to the naming scheme, any class SSHScan::A::B::C::D that requires configuration whitelisting can place a file `a_b_c_d.yml` in `configs`. This guarantees unique names for classes. (So SSHScan::Policy has `configs/policy.yml`)

I can think of a different scheme if you want.